packages/siop-oid4vp/lib/__tests__/IT.spec.ts

```diff
@@ -141,7 +141,7 @@ function getVCs(): IVerifiableCredential[] {
   return vcs
 }
 
-describe.skip('RP and OP interaction should', () => {
+describe('RP and OP interaction should', () => {
```

Comment:
If you are re-enabling tests, please remove the FIXME. But more importantly get in touch about what the FIXME/readme is about before re-enabling. UniResolvers are unstable in deployment. Hence the link to the ticket. By re-enabling this ticket we will again have tests failing every now and then

Reply:
Rolled it back to skip test